libibumad/umad.h

```diff
@@ -39,6 +39,7 @@
 #include <stdlib.h>
 #include <arpa/inet.h>
 #include <linux/types.h> /* __be16, __be32 and __be64 */
+#include <infiniband/verbs.h> /* union ibv_gid */
```

Comment:
I'm not sure we want to make the output of any configure process visible to libibumad users. Anyway, I will remove the #include directive.